### PR TITLE
fixed the styles for fees tooltip

### PR DESCRIPTION
**Pull request by Anchello**

### CODE REVIEW

File: packages/shared/components/fees-tooltip/fees-tooltip.scss [hunk 1]
```diff
@@ -13,11 +13,12 @@
   letter-spacing: 0.4px;
   font-weight: 400;
   @include font-size($font-size-paragraph);
+
   &__header,
   &__footer {
-    @include vertical-paddings($padding-small);
-    @include horizontal-paddings($padding-small);
+    padding: $padding-xsmall 25px;
```
Comment (pavelzubov) [inline packages/shared/components/fees-tooltip/fees-tooltip.scss]: Why?

File: packages/shared/components/fee-commission/fee-commission.scss [hunk 2]
```diff
@@ -8,7 +8,7 @@
   width: 100%;
   justify-content: space-between;
   &:not(:last-child) {
-    margin-bottom: $font-size-paragraph;
+    margin-bottom: 15px;
```
Comment (pavelzubov) [inline packages/shared/components/fee-commission/fee-commission.scss]: Use const

File: packages/shared/components/fees-tooltip/fees-tooltip.scss (hunk 1 above)
Comment (Anchello) [reply packages/shared/components/fees-tooltip/fees-tooltip.scss]: I think for mobile the padding inside this tooltip look very small
Снимок экрана 2019-07-03 в 16 22 48

Comment (pavelzubov) [reply packages/shared/components/fees-tooltip/fees-tooltip.scss]: Ok. Use for mobile another padding.

Comment (Anchello) [reply packages/shared/components/fees-tooltip/fees-tooltip.scss]: Done

File: packages/shared/components/fee-commission/fee-commission.scss (hunk 2 above)
Comment (Anchello) [reply packages/shared/components/fee-commission/fee-commission.scss]: Done

**Pull request merged**